### ISSUE TITLE
MBAR version

### ISSUE THREAD

**Issue by yunhuige** [vvoelz/nmr-biceps#8]
The current scripts only works well with pymbar 2.1.0b0 version and don't work with pymbar 3.0 (tested)! We may have to fix it or specify MBAR version for users.

**Comment by yunhuige** [comment]
I think I fixed this bug but I need to do more tests.

**Comment by yunhuige** [comment]
Updated:
I did tests using 2 different versions Pymbar (2.1.0 vs 3.0.3) and here is what I found:
our old scripts for this line is this:
(Deltaf_ij, dDeltaf_ij) = mbar.getFreeEnergyDifferences()
It works well with v2.1.0 but doesn't work with v3.0.3. To fix it, we need replace it with either of the following lines:
(Deltaf_ij, dDeltaf_ij, Theta_ij) = mbar.getFreeEnergyDifferences(uncertainty_method='svd-ew')
or
(Deltaf_ij, dDeltaf_ij, Theta_ij) = mbar.getFreeEnergyDifferences(uncertainty_method='approximate')
But I found either of these lines doesn't work with v2.1.0. 
So here is my suggestion: in the future we use the later MBAR scripts and we also ask people to use that version. We are going to abandon the old scripts and Pymbar v2.1.0.
